libraries/AP_Devo_Telem/AP_Devo_Telem.h

```diff
@@ -0,0 +1,73 @@
+// -*- tab-width: 4; Mode: C++; c-basic-offset: 4; indent-tabs-mode: nil -*-
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef __AP_DEVO_TELEM_H__
+#define __AP_DEVO_TELEM_H__
+
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Param/AP_Param.h>
+#include <AP_Math/AP_Math.h>
+#include <AP_GPS/AP_GPS.h>
+#include <AP_AHRS/AP_AHRS.h>
+#include <AP_Baro/AP_Baro.h>
+#include <AP_BattMonitor/AP_BattMonitor.h>
+#include <AP_SerialManager/AP_SerialManager.h>
+
+typedef uint8_t byte;
+
+#define DEVOM_SYNC_BYTE        0xAA
+
+#pragma pack(push, 1)
+typedef struct {
+        uint8_t header;                                ///< 0xAA for a valid packet
+        int32_t lon;
+        int32_t lat;
+        int32_t alt;
+        int16_t speed;
+        int16_t temp;
+        int16_t volt;
+        uint8_t crc8;   ///< CRC8
```

Comment:
This is used as a sum, not a crc. Does the spec label it as crc like this? I would suggest changing it to sum to prevent confusion.